.travis.yml

```diff
@@ -4,6 +4,7 @@ sudo: required
 
 os: linux
 dist: trusty
+group: edge
```

Comment:
Travis added [new images](https://blog.travis-ci.com/2017-06-19-trusty-updates-2017-Q2) for their Ubuntu Trusty builds! A comment would have helped...